code/drasil-lang/Language/Drasil/Chunk/Unitary.hs

```diff
@@ -41,12 +40,6 @@ unitary :: (IsUnit u) =>
 unitary i t s u space = UC (mkQuant i t s space (Just uu) Nothing) uu -- Unit doesn't have a unitDefn, so [] is passed in
   where uu = unitWrapper u
 
--- | For when the symbol changes depending on the stage
-unitary' :: (IsUnit u) =>
```

Comment:
Doesn't the removal of `unitary'` now make it impossible for a user to hand-stage a unitary?

Reply:
You're right - fixed in 3e8bf7863